include/zephyr/drivers/modem/simcom-sim7080.h

```diff
@@ -283,6 +283,15 @@ int mdm_sim7080_start_gnss(void);
  * @note The modem needs to be booted for this function to work.
  * Concurrent use of network and gnss is not possible.
  * @note If enabling xtra functionality fails a normal cold start will be performed.
+ * @warning On some modem revisions (e.g. 1951B16SIM7080) power cycling the gnss functionality
```

Comment:
Nit, but might be of interest to someone

```suggestion
 * @warning On some modem firmware revisions (e.g. 1951B16SIM7080) power cycling the gnss functionality
```